hidimstat/multi_sample_split.py

```diff
@@ -20,12 +31,27 @@ def aggregate_medians(list_sf):
 
 
 def aggregate_quantiles(list_sf, gamma_min=0.2):
-    """Aggregation of survival function values with the Meinshausen procedure
+    """Aggregation of survival function values by adaptive quantile procedure
 
     Parameters
     -----------
-        list_sf : ndarray or scipy.sparse matrix, (n_iter, n_features)
-            List of survival fuction values
+    list_sf : ndarray, shape (n_iter, n_features)
+        List of survival function values
```

Comment:
why do you call these 'survival function values' ? to me these are p-values ?

Reply:
It is not, it contains more information, but it is a bit harder to manipulate:
p-value = 0.05 ---> sf = 0.025 if effect is positive and sf = 0.975 if effect is negative.
p-value = 1.00 ---> sf = 0.5.